src/sentry/options/defaults.py

```diff
@@ -206,5 +206,8 @@
 register("nodedata.cache-sample-rate", default=0.0, flags=FLAG_PRIORITIZE_DISK)
 register("nodedata.cache-on-save", default=False, flags=FLAG_PRIORITIZE_DISK)
 
+# Use nodestore for eventstore.get_events
+register("eventstore.use-nodestore", default=False, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
Shouldn't this be registered in a separate PR

Reply:
Yup, https://github.com/getsentry/sentry/pull/16686